passworddialog.cpp

```diff
@@ -47,9 +57,18 @@ PasswordDialog::~PasswordDialog()
 {
 }
 
+void PasswordDialog::resizeEvent (QResizeEvent *event) {
```

Comment:
@palinek is more than right; manual eliding is wrong. Text eliding should be done by the Qt widget style, unless the whole style is created in the code, which isn't the case here.